config/config.go

```diff
@@ -261,6 +271,10 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("cache.host", "")
 	v.SetDefault("cache.query", "")
 	v.SetDefault("cache.expected_millis", 10)
+	v.SetDefault("cache.default_ttl_seconds.banner", 0)
```

Comment:
why are the defaults set to 0?

Reply:
a value of 0 means don't set the TTL in the cache request ... basically a TTL of 0 means no default is set.